services/web/src/screens/Settings/Security.js

```diff
@@ -1,371 +1,197 @@
-import React from 'react';
+import { useState } from 'react';
 import {
-  Segment,
+  Grid,
   Divider,
-  Message,
-  Header,
-  Form,
+  Title,
   Button,
-  Dimmer,
-  Loader,
-} from 'semantic';
+  Group,
+  LoadingOverlay,
+  Text,
+  Stack,
+  Fieldset,
+  ActionIcon,
+} from '@mantine/core';
 
-import { withSession } from 'stores/session';
+import { useSession } from 'stores/session';
 
 import Meta from 'components/Meta';
-import Layout from 'components/Layout';
+
 import ErrorMessage from 'components/ErrorMessage';
 import AppleDisableButton from 'components/Auth/Apple/DisableButton';
 import GoogleDisableButton from 'components/Auth/Google/DisableButton';
 
 import { createPasskey, removePasskey } from 'utils/auth/passkey';
 import { formatDate, fromNow } from 'utils/date';
 
-import { request } from 'utils/api';
-
 import Menu from './Menu';
+import Sessions from './components/Sessions';
```

Comment:
I'm not really in love with having `components` directories that aren't top level. I prefer more to put associated code into a new directory and helper components can just be in there (no need to point out that they are components, everything is a component essentially)

Reply:
fixed